README.md

```diff
@@ -50,18 +50,18 @@ The course's lesson material is broadly subdivided into three main topics:
 3. Reproducible science (Lectures 13-15)
     * Prepares students for project work period and introduces methods for reproducible science (GitHub, R Markdown).
 
+## Content Reuse Instructions
+
 If you are interested in using or modifying this content and repository for your 
-own course, you'll need to make a change to the Travis CI setup so you can host 
-the website on your own account. Check out the documentation on the [R Markdown
-online book](https://bookdown.org/yihui/bookdown/github.html). Here is a brief 
-excerpt taken from the book:
+own course, there are a few steps you need to take:
+
+1. Create a fork of this repository.
+1. Create a [personal access token](http://bit.ly/2cEBYWB) for your account on GitHub (make sure to enable the "repo" scope so that using this token will enable writing to your GitHub repos) and copy the token to your clipboard.
```

Comment:
Can we replace this with the direct link to https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line ? I don't see why we need a shortener and/or link tracking.